src/components/NewWorkspaceModal.ts

```diff
@@ -319,11 +319,19 @@ const NewWorkspaceModal = withDisplayName(
     };
 
     const isBillingProjectApplicable = (project: BillingProject): boolean => {
-      // As of AJ-1164, if requireEnhancedBucketLogging is true, then azure billing projects are ineligible.
-      // This coupling of enhanced bucket logging and billing project may change in the future.
-      if (!!workflowImport || !!requireEnhancedBucketLogging) {
+      // This is used when importing data to enforce a specific cloud.
+      if (cloudPlatform && project.cloudPlatform !== cloudPlatform) {
+        return false;
+      }
+      if (workflowImport) {
         return !isAzureBillingProject(project);
       }
+      // If creating a new workspace and enhanced bucket logging is required, allow all GCP projects
+      // (user will be forced to select "Workspace will have protected data" for GCP projects)
+      // and Azure billing projects that support protected Data.
+      if (!cloneWorkspace && requireEnhancedBucketLogging && isAzureBillingProject(project)) {
```

Comment:
This conditional logic was complicated before this change, and is even moreso now.   One of the friction points I have when reading for comprehension is having to invert/uninvert the cloneWorkspace boolean.

To simplify readability, would you consider adding and using a new local variable which will at least help some of the negative boolean inversion?

`const newWorkspace = !cloneWorkspace`?

Reply:
I updated the comment and it now makes more sense with the variable as `!cloneWorkspace`.